uptycs/manifest.json

```diff
@@ -43,6 +43,16 @@
     ]
   },
   "assets": {
+    "integration": {
+      "source_type_name": "Uptycs",
+      "configuration": {},
+      "events": {
+        "creates_events": true
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
```

Comment:
Please add a service_checks.json file to the assets directory. It can be a file that only includes [] though. See here for an example: https://github.com/DataDog/integrations-extras/blob/master/stormforge/assets/service_checks.json

Reply:
@mgarabed service_checks.json file was already added

Reply:
https://github.com/DataDog/integrations-extras/blob/master/uptycs/assets/service_checks.json

Reply:
Got it - thank you!